lib/constants/index.js

```diff
@@ -10,27 +10,47 @@
  * in the redux store in case
  * of collisions with other plugins
  */
```

Comment:
If we're going to keep all constants in a single file, does it need to be in a directory? Could it just be a single `constants.js` file?

Reply:
There are multiple files in the `constants` directory. Do you think it should be modularized more? There are some defined in `index`, but also others defined in the other files

Reply:
Yeah, I think so. I feel like indexes should either be fully an "entrypoint" or fully code logic. Feels easier to reason about.